k8s/release/admin/mail-relay/mailrelay.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: mailrelay
+  namespace: admin
```

Comment:
shouldn't this be in it's own namespace?

Reply:
A mail relay si quite an "adminy" thing ... but yeah I've been asking myself the same question.